src/integration/java/io/pinecone/integration/controlPlane/pod/CollectionTest.java

```diff
@@ -120,10 +110,17 @@ public void testIndexFromCollectionHappyPath() throws InterruptedException {
 
     @Test
     public void testCreateIndexFromCollectionWithDiffMetric() throws InterruptedException {
-        // Note collection == 1 index
-        String metricForNewIndex = IndexMetric.DOTPRODUCT.toString();
+        // Use a different metric than the source index
+        String[] metrics = {IndexMetric.COSINE.toString(), IndexMetric.EUCLIDEAN.toString(), IndexMetric.DOTPRODUCT.toString()};
```

Comment:
Added this back in since we're using whatever metric the index was created with in `TestIndexResourcesManager`.